src/ResourceManager/Automation/Commands.Automation/Cmdlet/ImportAzureAutomationRunbook.cs

```diff
@@ -66,6 +66,7 @@ public class ImportAzureAutomationRunbook : AzureAutomationBaseCmdlet
             Constants.RunbookType.PowerShellWorkflow,
             Constants.RunbookType.GraphicalPowerShellWorkflow,
             Constants.RunbookType.Graph,
+            Constants.RunbookType.Python,
```

Comment:
Might be a good idea to call this Python2 (the backend already makes the difference between Py2 and Py3).

Reply:
Hi Diastro,I thought the same, my only concern was there is no reference to the powershell version, Core or otherwise. So I assumed when the day came that Python 3 was supported, Python 2 would be deprecated. I can change it to Python2 before you accept the change. What does everyone else think?

Reply:
@Diastro gentle ping

I think we should also look into test coverage around this so that we will know immediately when Python 2 support is deprecated and we can move to Python 3.

Reply:
@cormacpayne are we doing integration testing with this module? How would you recommend detecting whether the API has deprecated Python2 or not?

Reply:
@rdbartram we have some live tests we run in Azure Automation, so we could add test coverage there to know when the call no longer works. We also should add a scenario test that runs during our regular CI to make sure this functionality is working correctly. @vrdmr @Diastro it doesn't look like we have any tests around the `Import-AzureRmAutomationRunbook` cmdlet, are we able to add one for this functionality?

Reply:
@rdbartram Sorry for the delay this slipped through my notifications.
I don't think we will deprecate Python2 if Python3 becomes available (we almost threat the as different languages to some extent). For now I think we can leave it as it (the variable name is really just an internal implementation detail at this point and can be refactored later). @vrdmr can comment on the test question.

Reply:
@cormacpayne I just synced with @vrdmr and you can open a task against @najams he's the owner for this area of our RP. We are waiting for our SDK to be migrated from Hyak to Swagger which will happen in December (this is when we will add test for this).
In the mean time can we let this go in please?